modules/create_timepoint/php/create_timepoint.class.inc

```diff
@@ -149,34 +136,24 @@ class Create_Timepoint extends \NDB_Form
         $visitLabelAdded    = false;
 
         foreach (\Utility::toArray($visitLabelSettings) as $visitLabel) {
-            if ($visitLabel['@']['subprojectID']==$this->subprojectID) {
-                if ($visitLabel['generation'] == 'user') {
-                    error_log(
-                        "DEPRECATION WARNING: User generated visit labels will ".
```

Comment:
what version was this deprecation warning added in? Have people had sufficient time to react?

Reply:
i think its been a couple years

Reply:
https://github.com/aces/Loris/pull/2451